decidim-proposals/spec/validators/proposal_length_validator_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe ProposalLengthValidator do
+  subject { validatable.new(body: body) }
+
+  let(:validatable) do
+    Class.new do
+      def self.model_name
+        ActiveModel::Name.new(self, nil, "Validatable")
+      end
+
+      include Decidim::AttributeObject::Model
```

Comment:
`Decidim::AttributeObject` doesn't exist in 0.26.

Maybe simply changing this to `Virtus.model` might work?